File: rtsp/src/main/java/com/pedro/rtsp/utils/ConnectCheckerRtsp.java
package com.pedro.rtsp.utils;

/**
 * Created by pedro on 20/02/17.
 */

public interface ConnectCheckerRtsp {

  void onConnectionSuccessRtsp();

  void onConnectionFailedRtsp(String reason);

  void onNewBitrateRtsp(long bitrate);

  void onDisconnectRtsp();

  void onAuthErrorRtsp();

  void onAuthSuccessRtsp();
}

File: rtsp/src/main/java/com/pedro/rtsp/utils/ConnectCheckerRtsp.kt
package com.pedro.rtsp.utils

/**
 * Created by pedro on 20/02/17.
 */
interface ConnectCheckerRtsp {
  fun onConnectionStartedRtsp(rtspUrl: String)
  fun onConnectionSuccessRtsp()
  fun onConnectionFailedRtsp(reason: String)
  fun onNewBitrateRtsp(bitrate: Long)
  fun onDisconnectRtsp()
  fun onAuthErrorRtsp()
  fun onAuthSuccessRtsp()
}